bin/distmap

```diff
@@ -774,22 +767,23 @@ OBSOLETE: picard is distributed in a single file now. Use --picard instead.
 
 =item B<--only-trim>
 
-This option does nothing. Trimming is now done on upload to the HDFS.
+OBSOLETE: This option does nothing. Trimming is now done on upload to the HDFS.
 
 
 =item B<--no-trim>
```

Comment:
It is now a bit unclear how to turn trimming off entirely. I assume it is empty --trim-args as we discussed, but maybe this should be mentioned somewhere.

Reply:
I just made it obsolete, but the behavior is still the same... should I really remove all obsolete behavior? In that case, it'll be a breaking change...

Reply:
Changed to DEPRECATED instead. It is still in use, and should pass `--no-trim` for skip trimming; otherwise it would use default values.